package/yast2-packager.spec

```diff
@@ -83,6 +83,8 @@ Recommends:     yast2-registration
 # force *-webpin subpackage removal at upgrade
 Obsoletes:      yast2-packager-webpin < %version
 
```

Comment:
To make clear that it has nothing to do with the `-webpin` thingie (although is obvious).